src/main/java/com/toopher/ToopherAPI.java

```diff
@@ -153,362 +151,145 @@ public ToopherAPI(String consumerKey, String consumerSecret, URI uri, HttpClient
 	    }
     }
 
-
-    /**
-     * Create a pairing
-     *
-     * @param pairingPhrase
-     *            The pairing phrase supplied by the user
-     * @param userName
-     *            A user-facing descriptive name for the user (displayed in requests)
-     * @return A PairingStatus object
-     * @throws RequestError
-     *             Thrown when an exceptional condition is encountered
-     */
-    public PairingStatus pair(String pairingPhrase, String userName) throws RequestError {
-        return this.pair(pairingPhrase, userName, null);
-    }
-
     /**
-     * Create a pairing
-     *
-     * @param pairingPhrase
-     *            The pairing phrase supplied by the user
+     * Create a QR pairing
```

Comment:
This is a big vague. I think we'll want to add some additional information, like "This return a PNG that should be scanned by the user to complete the pairing process."